cmd/agent/app/reporter/tchannel/reporter.go

```diff
@@ -65,6 +65,12 @@ func (r *Reporter) Channel() *tchannel.Channel {
 	return r.channel
 }
 
+// Close the underlying channel
+func (r *Reporter) Close() error {
+	r.channel.Close()
```

Comment:
same comment as for grpc connection - the reporter is not the only user of the channel, so it cannot be in charge of closing it.